melos.yaml

```diff
@@ -20,8 +20,11 @@ ide:
 
 scripts:
   format: dart format --fix --line-length 120 .
-  format-check: dart format --output=none --set-exit-if-changed --line-length 120 .
+  format:check: dart format --output=none --set-exit-if-changed --line-length 120 .
   analyze: dart analyze --fatal-infos .
   test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test --concurrency=$(nproc --all)"
+  test:changed: |
+    packages_glob="{$(echo $(melos list --parsable --diff=origin/main --include-dependents | xargs -r -n1 basename) | sed "s/ /,/g")}"
+    melos exec --scope="$packages_glob" --concurrency=1 --fail-fast --dir-exists=test -- "flutter test --concurrency=$(nproc --all)"
```

Comment:
Does it make sense to move this into it's own shell script with proper variables and methods?

It's kinda hard to understand

Reply:
Yeah, I'm struggling with this right now as it gets quite bad...